nms/packages/magmalte/fbc_js_core/ui/components/ProfileButton.js

```diff
@@ -106,10 +95,14 @@ const ProfileButton = () => {
     location.pathname.includes(adminUrl) ||
     location.pathname.includes(settingsUrl);
 
+  const hasAccountSettings = !ssoEnabled;
```

Comment:
This looks like something that could potentially be re-used. Could we move this into `AppContext`?